docs/DeveloperGuide.adoc

```diff
@@ -429,12 +436,115 @@ _{ more test cases ... }_
 .. Other incorrect delete commands to try: `delete`, `delete x` (where x is larger than the list size) _{give more}_ +
    Expected: Similar to previous.
 
```

Comment:
Please update the deleting a person developer guide. Our delete is not by id is by transaction_id